apps/condo/domains/ticket/components/TicketId/TicketDeferredDateField.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from 'antd'
+import dayjs from 'dayjs'
+import React from 'react'
+
+import { useIntl } from '@core/next/intl'
+import { Ticket, TicketStatusTypeType } from '@app/condo/schema'
```

Comment:
do we need this `TypeType` really?

Reply:
`ticket.status.type` is of type `TicketStatusTypeType`

how can it be done otherwise?

Reply:
It's an autogenerated name :)
<img width="912" alt="image" src="https://user-images.githubusercontent.com/25384290/184078542-e70fcca2-c087-4fbd-8a98-6bac86a7bccc.png">